### DISCUSSION TITLE
Dịch 'generator' và 'discriminator'

### DISCUSSION THREAD

**Discussion by minhduc0711** [d2l-ai/d2l-vi#2822] [Terminology]
Từ này @trGiang99 đang để là `bộ sinh` và `bộ phân biệt`. Mình thấy ổn, mọi người có ý kiến gì không nhỉ?

cc: @lkhphuc @rootonchair @cuongvng @davidnvq

**Comment by lkhphuc** [comment]
Sẵn tiện mọi người dịch chương gan có nhắc tới cụm `generative/discriminative model(ling)`?
modelling
`model(ling)` là `mô hình (hoá)`
vậy
`discriminative model(ling)` => `mô hình (hoá) phân biệt` và
`generative model` -> `mô hình sinh` thì ok còn
`generative modelling` => `mô hình hoá sinh` à 😂

**Reply by cuongvng** [reply]
thêm chữ `hóa` đọc ra `biochemical model` :v.
`sự/việc mô hình sinh` thì sao anh?

**Reply by trgiangdo** [reply]
Em nghĩ với `modeling` thì tạm thời thêm chữ `sự` vào cho rõ nghĩa. 
`mô hình hoá sự phân biệt` và `mô hình hoá sự sinh`.

Hoặc `mô hình hoá quá trình phân biệt` và `mô hình hoá quá trình sinh`, nhưng dùng `quá trình` thì không biết có diễn tả đúng trong ngữ cảnh này không.
Cái này thì nhờ các anh phân tích :D

**Reply by rootonchair** [reply]
`Mô hình hóa tác vụ phân biệt` và `mô hình hóa tác vụ sinh` thì sao?
